doc/reference/storage_drivers.md

```diff
@@ -16,6 +16,7 @@ storage_cephobject
 storage_ceph
 storage_dir
 storage_lvm
+storage_powerflex
```

Comment:
We're sorting these alphabetically by title, so "Dell PowerFlex" should go before the dir driver. Or can we use just "PowerFlex" in the title and then specify that it's "Dell PowerFlex" later?

Reply:
Oh sorry - I'm talking about the documentation output, not the list here (but of course the list here controls what we get in the output ;) ).
Since the title of the page is "Dell PowerFlex", it should go before `storage_dir` here. Having `storage_powerflex` as the ID is totally fine.

Reply:
Ah ok I see, thanks, fixed :)